index.src.html

```diff
@@ -1103,11 +1103,19 @@ <h5 id="interfaces-credentialrequestoptions">
     <dd>
       If <code>true</code>, the user agent will only attempt to provide
       a {{Credential}} without user interaction. It MUST NOT present the user
-      with any visible prompt to grant access to a {{Credential}}: if the user
-      has opted-into always giving a particular site access to a particular set
-      of credentials, they will be provided. If not, the promise will resolve
-      with <code>undefined</code>. For processing details, see the algorithm
-      defined in [[#request-credential]].
+      with any visible prompt to grant access to a {{Credential}} or to select
+      a {{Credential}}: if the user has opted-into always giving a particular
```

Comment:
If you have one credential stored that requires user mediation, the dialog pops up to grant access. If you have two credentials stored, the dialog pops up to select the one you want to use. Makes sense or should I rephrase this?